### PR TITLE
[NCL-8119] Build config Denpendencies, Dependents and Group Configs tab

### PR DESCRIPTION
**Pull request by DnsZhou**

### CODE REVIEW

**Comment by DnsZhou** [comment]
all fixed.

**Comment by patrikk0123** [comment]
It's not "all fixed" when the pipeline tests fail..

**Comment by DnsZhou** [comment]
Hi @patrikk0123 , sorry about the missed imports. looks passed now.

File: src/components/BuildConfigDependantsPage/BuildConfigDependantsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <TextContent>
+            <Text component={TextVariants.h2}>Dependants</Text>
+          </TextContent>
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop shadow={false}>
```
Comment (patrikk0123) [inline src/components/BuildConfigDependantsPage/BuildConfigDependantsPage.tsx:35]: Remove this content box and put `borderBottom` on the toolbar above.

File: src/components/BuildConfigDependenciesPage/BuildConfigDependenciesPage.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          </TextContent>
+        </ToolbarItem>
+        <ToolbarItem>
+          <ProtectedComponent>
+            <ActionButton link="edit">Edit</ActionButton>
+          </ProtectedComponent>
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop shadow={false}>
```
Comment (patrikk0123) [inline src/components/BuildConfigDependenciesPage/BuildConfigDependenciesPage.tsx:41]: Remove this content box and put `borderBottom` on the toolbar above.

File: src/components/BuildConfigGroupConfigsPage/BuildConfigGroupConfigsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <TextContent>
+            <Text component={TextVariants.h2}>Group Configs</Text>
+          </TextContent>
+        </ToolbarItem>
+      </Toolbar>
+      <ContentBox borderTop shadow={false}>
```
Comment (patrikk0123) [inline src/components/BuildConfigGroupConfigsPage/BuildConfigGroupConfigsPage.tsx:34]: Remove this content box and put `borderBottom` on the toolbar above.

**Pull request merged**